Index: app/src/main/java/com/example/curruaapp/api/StoreApi.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/curruaapp/api/StoreApi.kt b/app/src/main/java/com/example/curruaapp/api/StoreApi.kt
--- a/app/src/main/java/com/example/curruaapp/api/StoreApi.kt
+++ b/app/src/main/java/com/example/curruaapp/api/StoreApi.kt	(date 1760287336073)
@@ -38,6 +38,6 @@
     @Multipart
     @POST("upload/image")
     suspend fun uploadImages(
-        @Part parts: List<MultipartBody.Part>
-    ): List<ImageResource>
+        @Part parts: MultipartBody.Part
+    ): ImageResource
 }
\ No newline at end of file
